### PR TITLE
feat(common): add logoURI prop to token interface

### PR DESCRIPTION
**Pull request by LuPoYi**
- [x] add logoURI(optional) prop to token interface
- [x] rename zksync USDC to USDC.e
- [x] yarn changeset

### CODE REVIEW

File: packages/common/src/tokens/token.ts [hunk 1]
```diff
@@ -16,8 +17,9 @@ export class Token {
   readonly decimals: number;
   readonly symbol: string;
   readonly name: string;
+  readonly logoURI: string | undefined;
```
Comment (zodahu) [inline packages/common/src/tokens/token.ts]:
```suggestion
  readonly logoUri?: string;
```

Simplify it and currently we use camelCase for naming, such as `chainId`, `protocolId` and `lstApy`.

File: packages/common/src/tokens/token.ts
Comment (zodahu) [inline packages/common/src/tokens/token.ts:1]: 1. In the `isTokenObject` function, if `logoURI` exists, we can verify that its type is a string.
2. Could we include a test for the newly added `logoURI` field in `tokentest.ts`?

Comment (LuPoYi) [reply packages/common/src/tokens/token.ts:1]: fixed!

File: packages/common/src/tokens/token.ts (hunk 1 above)
Comment (LuPoYi) [reply packages/common/src/tokens/token.ts]: fixed

**Comment by zodahu** [comment]
LGTM

**Pull request merged**